helm/dagster/values.yaml

```diff
@@ -69,6 +69,11 @@ dagsterWebserver:
     # must be empty.
     externalConfigmap: ~
 
+    # Defines if the external configmap is optional. If set to true,
+    # the configmap will have to exist before deploying the helm chart
+    # default: false
+    externalConfigmapOptional: false
```

Comment:
Isn't it the other way around? If this is false, the optional field of the configmap in the podspec is set to false and the config map must preexist? what am I missing?

Reply:
Yes that is correct apologies, I will update it